ArduCopter/Parameters.h

```diff
@@ -162,10 +162,11 @@ class Parameters {
         //
         // 80: Heli
         //
+        k_param_heli_servo_4 = 79,
         k_param_heli_servo_1 = 80,
         k_param_heli_servo_2,
         k_param_heli_servo_3,
-        k_param_heli_servo_4,
```

Comment:
data in k_param_heli_servo_4 will be lost/corrupted, and posisble auto-populated into servo_yaw thus never allowing the defaults to be loaded. Better to create the _servo_yaw at 79. Also, please either change or remove the // 80: Heli label.